### PR TITLE
INTYGFV-15923: Simplified converter for certificate data. New value type for questions for mapping lists of item. 

### PR DESCRIPTION
**Pull request by mww-sopra-steria**

### CODE REVIEW

File: app/src/test/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactoryTest.java [hunk 1]
```diff
@@ -52,4 +54,27 @@ void shouldReturnHTMLForTable() {
...
+                .label("label 2")
+                .value("value 2")
+                .build()
+        ))
+        .build();
+
+    final var result = CertificateQuestionValueHTMLFactory.itemList(value);
+
+    assertEquals(
+        "<p><h5 className=\"ids-heading-5\">label 1</h5><p>value 1</p><h5 className=\"ids-heading-5\">label 2</h5><p>value 2</p></p>",
```
Comment (mhornfeldt) [inline app/src/test/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactoryTest.java:77]: From the example HTML I don't think it was a <p> around the question

File: app/src/main/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactory.java [hunk 2]
```diff
@@ -21,4 +27,16 @@ public static String table(CertificateQuestionValueTable question) {
   public static String list(CertificateQuestionValueList question) {
     return HTMLListFactory.ul(question.getValues());
   }
+
+  public static String itemList(CertificateQuestionValueItemList question) {
+    return p(
+        fromList(
+            question.getValues(),
+            valueItem -> join(
+                h5(valueItem.getLabel()),
```
Comment (mhornfeldt) [inline app/src/main/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactory.java:36]: I think it's supposed to be h4 according to figma

**Review by mhornfeldt — APPROVED** [review]
Nice :) Two comments about matching the example HTML

File: app/src/test/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactoryTest.java (hunk 1 above)
Comment (mww-sopra-steria) [reply app/src/test/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactoryTest.java]: Fixed!

File: app/src/main/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactory.java (hunk 2 above)
Comment (mww-sopra-steria) [reply app/src/main/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactory.java]: Fixed!

**Pull request merged**